core/src/main/java/edu/wpi/grip/core/serialization/Project.java

```diff
@@ -88,8 +95,27 @@ public void initialize(StepConverter stepConverter,
     return file;
   }
 
+  /**
+   * Sets the current project file to the given optional one. This also updates a file on disk
+   * so the app can "remember" the most recent save file when it's opened later.
+   *
+   * @param file the optional file that this project is associated with.
+   */
   public void setFile(Optional<File> file) {
+    file.ifPresent(f -> logger.info("Setting project file to: " + f.getAbsolutePath()));
     this.file = file;
+    try {
+      if (file.isPresent()) {
+        Files.write(file.get().getAbsolutePath(),
+            GripFileManager.LAST_SAVE_FILE,
+            Charset.defaultCharset());
+      } else {
+        // No project file, delete the last_save file
+        GripFileManager.LAST_SAVE_FILE.delete();
```

Comment:
Basically, the `LAST_SAVE_FILE` stores the full path to the most recently loaded save file. If no file was set, then the `LAST_SAVE_FILE` is deleted to prevent auto-loading a stale file.